config/config.go

```diff
@@ -158,3 +162,15 @@ func getFileHash(file []byte) (fileHash string, err error) {
 
 	return hex.EncodeToString(hash.Sum(nil)), nil
 }
+
+func validateConfig(config Config) error {
+	if !config.ExecutorConfig.UseRegoPolicy && config.ExecutorConfig.ExecutionMode == exConfig.PassthroughExecutionMode {
```

Comment:
I'm a bit confused on this logic. If you have pass through mode enabled, why would enforce there to be rego policy provider? Isn't pass through mode just consolidating the verifier reports and returning to GK?

Reply:
good question! The passthrough mode is supposed to work with rego policy provider only. The current config policy provider is incompatible with the passthrough mode.